app/scripts/controllers/transactions.js

```diff
@@ -956,7 +956,7 @@ angular.module('openhimConsoleApp')
 
         Api.Transactions.query(filters, function(transactions) {
           lastPollingCompleted = true;
-          transactions.forEach(function(trx) {
+          transactions.reverse().forEach(function(trx) {
```

Comment:
I wonder if we should reverse the transactions or rather sort by date. Not sure we will always get the transaction back in this order if we refactor unless we build in a guarantee for this in the API.

Reply:
The transactions shown in the log on initial load are also dependent on the order that the API returns them in. I guess the correct solution would be to add a sort query parameter to the endpoint and handle it in the core.

Reply:
Yeah ok, I agree. If that's the case we should add a guarantee to the API.

Reply:
Ah, I see we already do this: https://github.com/jembi/openhim-core-js/blob/master/src/api/transactions.coffee#L195

Ignore me :)

Reply:
Oh, yes I knew that was there but wasn't sure what you meant by adding a guarantee 👍